src/file_controller/mod.rs

```diff
@@ -267,8 +315,10 @@ impl Cache {
     }
 
     fn make_line_result(&self, file_path: &Path, span: &Span) -> Result<LineResult, String> {
+        use file_controller::Highlighted;
```

Comment:
Why do we need this import? Aren't we in `file_controller` here?

Reply:
This was just a dumb mistake, I'm surprised I didn't get an unnecessary import warning.